<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1984</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>511</width>
      <height>701</height>
     </rect>
    </property>
    <property name="title">
     <string>Large Language Model</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>481</width>
       <height>641</height>
      </rect>
     </property>
     <property name="title">
      <string>OpenAI Options</string>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>341</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>API Key</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="openaiAPIKey"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>151</width>
        <height>12</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
      <widget class="QSlider" name="temperatureOpenAI">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Max-tokens</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="maxTokenOpenAI">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="resetContextOpenAI">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>600</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Context</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testConnectionOpenAI">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Connection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetDefaultsOpenAI">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>600</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Defaults</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>System Prompt</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="systemPromptEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_context">
       <property name="text">
        <string>Context</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="contextBrowserOpenAI"/>
     </widget>
     <widget class="QCheckBox" name="chatModeOpenAI">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate Chat Mode</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxChat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QSplitter" name="splitter_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>401</width>
         <height>81</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>User Input</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="userInputOpenAI"/>
      </widget>
      <widget class="QPushButton" name="sendUserInputOpenAI">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="llmMBOX">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>164</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>LLM</string>
      </property>
     </widget>
     <widget class="QComboBox" name="LLMChocie"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>671</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>LLAMA Options</string>
    </property>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>341</width>
       <height>24</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>151</width>
       <height>12</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>70</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Max-tokens</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4096</number>
      </property>
      <property name="value">
       <number>1024</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>System Prompt</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="systemPromptEditLlama">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>301</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="testConnectionLlama">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Connection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetDefaultsLlama">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Defaults</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>User Input</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="userInputLlama"/>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate Chat Mode</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1984</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
